### ISSUE TITLE
GraphQL on microservices?

### ISSUE THREAD

**Issue by henriqueweiand** [Folkloreatelier/laravel-graphql#175]
I'm trying to implement a micro services schema for with laravel-graphql where I have for example two services:

- Users
- Enrollment

The user service is a laravel structure with its own database where only the people register.

The enrollment service is a laravel structure with its own bank where it has the register of the class and the link of the class to a student (foreign key).

Are distinct projects, with distinct but related bases, is it possible with graphQL in the user service to create a relationship in the model from which to search the information in a registration endpoint?

**Comment by SonarSoftware** [comment]
Definitely. You are implementing your own resolution mechanism, so you can load data from wherever you want. I don't use Eloquent/SQL at all in my resolvers. The resolution could be a call to a remote API, read from a text file, or whatever makes sense for your use case.
